database/tasks.js

```diff
@@ -1,23 +1,22 @@
 var express = require('express');
 var router = express.Router();
-const Queries = require( './queries.js' )
+const queries = require('./queries.js')
 const pgPromise = require('pg-promise')
 const pgp = pgPromise()
 const db = pgp({database: 'doitnow'})
 
-const Tasks = {
+const tasks = {
   getAll: () => {
-    return db.any( Queries.allTasks )
+    return db.any( queries.allTasks )    
   },
   create: ( name ) => {
-    return db.oneOrNone ( Queries.createTask, [name] )
+    return db.oneOrNone ( queries.createTask, [name] )
   },
   update: ( id, name ) => {
-    return db.one ( Queries.updateName [id, name] )
+    return db.one ( queries.updateName [id, name] )
   },
-  deleteTask: ( id ) => {
-    return db.none ( Queries.deleteTask [id] )
-  }
+  deleteTask: ( id ) => db.none ( queries.deleteTask, [id] )
```

Comment:
you can take out returns from other task functions just like you did here on delete since they are all one line return statements